src/App.tsx

```diff
@@ -17,18 +48,38 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                onFilterChange={handleFilterChange}
+                filter={filter}
+                inputValue={inputValue}
+                onInputChange={handleInputChange}
+                clearInput={clearInput}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {!todos.length
```

Comment:
It's a bad idea. Create state for `isLoading`  and change it before and after you make API call
Because now, when you will get empty response you will see loader all the time